src/Mesh/babylon.abstractMesh.ts

```diff
@@ -1185,6 +1183,7 @@
                         this._localWorld.multiplyToRef(this.parent.getWorldMatrix(), this._worldMatrix);
                     }
                 }
+                this._markSyncedWithParent();

```

Comment:
markSynced is for dont recompute.
you can add print here. and you'll see it never stop recompute.
even all meshes not moving.
I use an older version. but i think not big difference.